fastlane.gemspec

```diff
@@ -81,4 +81,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rest-client', '~> 1.6.7'
   spec.add_development_dependency 'fakefs', '~> 0.8.1'
   spec.add_development_dependency 'sinatra', '~> 1.4.8'
+  spec.add_development_dependency 'timecop'
```

Comment:
Is there an advantage to using it though? The only thing it does for us is mock out `Time.now` to return a `Time.utc(0)` object. Would something like e.g. `expect(Time).to receive(:now).and_return(Time.utc(0))` not work (and even be simpler to understand)?